check-setup-mds.log

```diff
@@ -0,0 +1,131 @@
+# MDS setup check 1.0.0
```

Comment:
I don't think we need this file in the repo since we have pasted an example of the output at the bottom of the installation instructions. Same with the PDF below.

Reply:
I agree, removed it.